qemu/tests/cfg/qmp_command.cfg

```diff
@@ -31,12 +31,11 @@
             cmd_return_value = "{u'status': u'paused', u'singlestep': False, u'running': False}"
         - qmp_query-name:
             qmp_cmd = "query-name"
-            post_cmd = "info name"
-            cmd_result_check = m_in_q
+            cmd_result_check = contain 
         - qmp_query-uuid:
+            uuid = 990ea161-6b67-47b2-b803-19fb01d30d12 
```

Comment:
Any strategy for using this value. or just a random value ?

Reply:
uuid just a random value for verify the result of "query-uuid"